syntax = "proto3";

package response_models;

message KilledPlayerResponseModel {
	int32 player_id = 1;
	int32 killed_by = 2; // id of player that made kill
}